servers/physics_2d/body_pair_2d_sw.cpp

```diff
@@ -58,6 +58,8 @@ void BodyPair2DSW::_contact_added_callback(const Vector2 &p_point_A, const Vecto
 	contact.acc_tangent_impulse = 0;
 	contact.local_A = local_A;
 	contact.local_B = local_B;
+	contact.A_body_contact_idx = -1;
+	contact.B_body_contact_idx = -1;
```

Comment:
These are only needed, because the `impulse` variable is not being updated in the `add_contact()` function.